### PR TITLE
EN Create Enterprise File Storage restore Vovlume using revert snapshot api

### PR DESCRIPTION
**Pull request by agoude**

### CODE REVIEW

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md [hunk 1]
```diff
@@ -27,106 +24,120 @@ Snapshots are created from an existing, operational volume. They cannot exist wi
...
 
+### Scenario 2: Reverting a volume using a snapshot taken via the snapshot policy
 
-## Scenario 2: Revert volume using snapshot taken via snapshot policy
 In this case, a snapshot policy takes regular (automatic) snapshots of a volume. You want to restore your volume to the latest snapshot taken by the policy.
 
-You hold the latest snapshot taken by the snapshot policy associated to a volume. This snapshot becomes `manual`. Once the snapshot has the type `manual`, its volume can be restored to it.
-
-### Requirements
+You will have to hold the latest snapshot taken by the snapshot policy associated to a volume. This snapshot becomes `manual`. Once the snapshot has the type `manual`, its associated volume can be restored to it.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md]: > This snapshot becomes `manual`.

This snapshot will become a `manual` snapshot. 

I would change the sentence above to match the `You will have...` from the previous sentence and for more clarity perhaps too.

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md [hunk 2]
```diff
@@ -27,106 +24,120 @@ Snapshots are created from an existing, operational volume. They cannot exist wi
...
-2\.	Hold the Snapshot using the following API route: 
->[!api]
+2\. Hold the Snapshot using the following API call: 
+
+> [!api]
 >
 > @api {POST} /storage/netapp/{serviceName}/share/{shareId}/snapshot/{snapshotId}/hold
+
 - `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md:110]: - `{snapshotId}` is the latest `automatic` snapshot

`{snapshotId}` path parameter description is lacking here after `{shareId}`

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md [hunk 3]
```diff
@@ -27,106 +24,120 @@ Snapshots are created from an existing, operational volume. They cannot exist wi
...
 >
-- `{serviceName}` is the service unique ID 
+
+- `{serviceName}` is the service unique ID
 - `{shareId}` is the share to restore
-- `{snapshotID}` is the latest share snapshot 
+- `{snapshotID}` is the latest share snapshot
 
-The OVHcloud API will return a HTTP 202 (Accepted) and no response body.
+The OVHcloud API will return a, HTTP 202 (Accepted) code and no response body.<br>
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md]: > The OVHcloud API will return a, HTTP 202 (Accepted) code and no response body.<br>

 The OVHcloud API will return **an** HTTP 202 (Accepted) code and no response body.<br>

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md (hunk 1 above)
Comment (OVH-Team-Guides) [reply pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md]: Thanks @dariakts , I've adjusted the wording, let me know if that's better.

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md (hunk 2 above)
Comment (OVH-Team-Guides) [reply pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md:110]: Added, thanks

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md (hunk 1 above)
Comment (dariakts) [reply pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.en-gb.md]: It's all good for me 👍 @OVH-Team-Guides

**Comment by OVH-Team-Guides** [comment]
@agoude  and @dariakts , please review the EN>FR translation

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+## Objectif
+
+Dans ce guide, nous allons expliquer comment restaurer un volume à son dernier snapshot, à l'aide de la fonction de *snapshot revert*.
+
+**Apprenez à restaurer les volumes de votre solution Enterprise File Storage à l'aide de la fonctionnalité de restauration de snapshots via l'API OVHcloud.**
+
+## Prérequis
+
+- Un service Enterprise File Storage OVHcloud avec un volume disponible
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
- Disposer d'une offre OVHcloud Enterprise File Storage avec un volume
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+- Être connecté à l’[API OVHcloud](https://api.ovh.com/)
+
+## Principes de base
+
+Un snapshot de volume est une copie ponctuelle en lecture seule d'un volume.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
Un snapshot (ou instantané) d'un volume est une copie ponctuelle en lecture seule d'un volume.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+Les snapshots sont créés à partir d'un volume opérationnel existant. Ils ne peuvent exister sans cela.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
Les snapshots sont crées à partir d'un volume existant et opérationnel. Un snapshot ne peut pas être utilisé si le volume auquel il appartient n'existe plus.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+> [!warning]
+>
+> Veuillez noter qu'une fois qu'un volume est restauré à l'aide d'une copie de snapshot, tous les fichiers ou snapshots créés ultérieurement seront perdus. Lorsqu'un volume est restauré, toutes les données qu'il contient sont remplacées par les données du snapshot. Cette action est irréversible.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
> Veuillez noter qu'une fois qu'un volume est restauré à l'aide d'un snapshot, tous les fichiers ou snapshots créés ultérieurement seront perdus. Lorsqu'un volume est restauré, toutes les données qu'il contient sont remplacées par les données du snapshot. Cette action est irréversible.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+## En pratique
+
+### Scénario 1 : restaurer un volume depuis un snapshot de type `manual`
+
+Dans ce scénario, vous souhaitez restaurer votre volume à son dernier snapshot de type `manual`, créé via l'API OVHcloud ou l'espace client OVHcloud.
+
+> [!primary]
+> **Prérequis pour ce scénario :**
+>
+> - Vous avez déjà effectué un snapshot de type `manual`. Dans le cas contraire, vous pouvez créer un snapshot de type `manual` via l’API OVHcloud ou votre espace client OVHcloud.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
> - Vous avez déjà creé un snapshot de type `manual`. Dans le cas contraire, vous pouvez créer un snapshot de type `manual` via l’API OVHcloud ou votre espace client OVHcloud.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+> - Le snapshot de type `manual` doit appartenir au volume que vous souhaitez restaurer.
+
+1\. Identifiez le dernier snapshot de volumed e type `manual` à l'aide de l'appel API suivant :
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
1\. Identifiez le dernier snapshot de type `manual` à l'aide de l'appel API suivant :
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+2\. Restaurez votre volume à son dernier snapshot en utilisant l'appel API `/revert` : 
+
+> [!api]
+>
+> @api {POST} /storage/netapp/{serviceName}/share/{shareId}/revert
+>
+
+- `{serviceName}` est l'identifiant unique du service
+- `{shareId}` est le volume à restaurer
+- `{snapshotID}` est le dernier snapshot de volume
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
- `{snapshotID}` est le dernier snapshot du volume
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+
+L’API OVHcloud retournera uniquement un code HTTP 202 (*Accepted*).<br>
+L'état du volume passera à `reverting`, puis reviendra à `available` une fois le processus de restauration du volume terminé. De manière simultanée, l'état du snapshot passera à `restoring` puis reviendra à `available` une fois le processus de restauration du volume terminé.
+
+![RevertManualSnapshot](images/use_case_1_step_2.png){.thumbnail}
+
+### Scénario 2 : Restaurer un volume depuis un snapshot effectué via la Snapshot policy
+
+Dans ce scénario, une règle de snapshots (*Snapshot policy*) prend des snapshots réguliers (automatiques) d'un volume et vous souhaitez restaurer votre volume au dernier snapshot pris par la stratégie.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
Dans ce scénario, une règle d'une politique de snapshots (*Snapshot policy*) prend des snapshots réguliers (automatiques) d'un volume et vous souhaitez restaurer votre volume au dernier snapshot crée par la *Snapshot policy*.
```

Snapshot policy is translated to *une politique de snapshots* inside [this FR guide](https://help.ovhcloud.com/csm/fr-public-cloud-storage-netapp-snapshot-policies?id=kb_article_view&sysparm_article=KB0059123). A Snapshot Policy can have multiple `rules / régles`, each one taking snapshots at a scheduled interval.

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+
+> [!primary]
+> **Prérequis pour ce scénario :**
+>
+> - Vous avez créé une *snapshot policy* et vous l'avez associée au volume à restaurer.
+> - Cette *snapshot policy*  a créé au moins un snapshot.
+
+> [!primary]
+>
+> Les snapshots pris par la *snapshot policy*  sont de type `automatic`. Pour qu'ils puissent être utilisés pour la restauration de volume, ils doivent être conservés en utilisant la route API `/hold`. Cela empêchera leur rotation par la planification de la Snapshot Policy.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
> Les snapshots pris par la *snapshot policy*  sont de type `automatic`. Pour qu'ils puissent être utilisés pour la restauration de volume, ils doivent être conservés en utilisant la route API `/hold`. Cela empêchera leur rotation par la *Snapshot policy*.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,150 @@
...
+>
+
+1\. Identifiez le dernier snapshot de volume de type `automatic` à l'aide de l'appel API suivant :
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
1\. Identifiez le dernier snapshot de type `automatic` à l'aide de l'appel API suivant :
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+> Une fois l'opération de conservation (`hold`) effectuée, l'identifiant et le type du snapshot seront modifiés. Toutefois, ses propriétés `name`, `createdAt` et `path` seront conservées. Veuillez prendre note du nouvel `id` pour les étapes suivantes.
+>
+
+![RevertSnapshot](images/use_case_2_step_2.png){.thumbnail}
+
+3\. Assurez-vous que le nouveau snapshot est le dernier snapshot de type `manual` du volume.
+
+Si d'autres snapshots de type `manual` ont été pris avant ce snapshot, ils devront être supprimés.
+
+4\. Le point de terminaison utilisé pour récupérer la liste des snapshots de volumes de l'étape 1 peut être réutilisé ici.
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
4\.  La route API utilisée pour récupérer la liste des snapshots du volume de l'étape 1 peut être réutilisée ici.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md [hunk 15]
```diff
@@ -0,0 +1,150 @@
...
+5\. Restaurez le volume à son dernier snapshot en appelant la route API `/revert` :
+
+> [!api]
+>
+> @api {POST} /storage/netapp/{serviceName}/share/{shareId}/revert
+>
+
+- `{serviceName}` est l'identifiant unique du service
+- `{shareId}` est la volume à restaurer
+- `{snapshotID}` est le dernier snapshot de volume en date
```
Comment (dariakts) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_revert_snapshot/guide.fr-fr.md]:
```suggestion
- `{snapshotID}` est le dernier snapshot en date du volume
```

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @agoude 
Translations will come soon

**Pull request merged**